README.rst

```diff
@@ -308,7 +308,7 @@ How they are selected and govern is outlined in our `governance document <https:
 Adopters
 --------
 A list of adopters of the Cilium project who are deploying it in production, and of their use cases,
-can be found in file `USERS.md <https://raw.githubusercontent.com/cilium/cilium/main/USERS.md>`__.
+can be found in file `USERS.md <https://github.com/cilium/cilium/blob/main/USERS.md>`__.
```

Comment:
The commit message was to fix link, but the existing link is working as expected, probably it's about using github.com link instead, then I am not sure about the value proposition for this change :thinking:

Reply:
@sayboras The `raw.githubusercontent.com` link results in a raw markdown document displayed in the browser, while the new link proposed here renders the markdown displayed through the GH UI. Much nicer IMO.